packages/lab/src/parent-child-layout/ParentChildLayout.css

```diff
@@ -1,30 +1,30 @@
 .saltParentChildLayout {
-  overflow: hidden;
+  display: flex;
+  gap: calc(var(--parentChildLayout-gap) * 3);
 }
 
-.saltParentChildLayout-parent {
-  animation: var(--salt-animation-slide-in-left);
+.saltParentChildLayout-child {
+  flex-grow: 1;
```

Comment:
Child will expand to fill the rest of the container - can be seen in the preferences dialog example.
Think this makes the most sense if the parent is a fixed width or leaving empty space. Use can add styling to align centre or fix the total width of the ParentChildLayout if preferred